src/config/appConfig.js

```diff
@@ -63,7 +63,7 @@ const conf = {
 
   /** dev-option:
    * skips the QR login after 500ms */
-  automateQrLogin: __DEV__ && false,
```

Comment:
I wouldn't mind leaving that one as it is. But the default was `false` so far.
Was that on purpose?

Reply:
No, it wasn't😅